backend/apps/api/tests/conftest.py

```
@@ -38,4 +74,15 @@ def mock_s3_client():
 
 def pytest_sessionfinish(session, exitstatus):
```

Ruff (0.14.1)

Unused function argument: `session`

(ARG001)